org.jacoco.doc/docroot/doc/repo.html

```diff
@@ -48,41 +48,71 @@
 <h2>Artifacts</h2>
 
 <p>
-  JaCoCo is split into several separate projects, the artifact ids are:   
+  Following JAR files are available:
 </p>
 
 <table class="coverage">
   <thead>
     <tr>
       <td>Group ID</td>
       <td>Artifact ID</td>
+      <td>Classifier</td>
       <td>Description</td>
     </tr>
   </thead>
   <tbody>
     <tr>
       <td><code>org.jacoco</code></td>
       <td><code>jacoco-maven-plugin</code></td>
-      <td>Plug-in for Maven builds</td>
+      <td></td>
+      <td>Plug-in for Maven</td>
     </tr>
     <tr>
       <td><code>org.jacoco</code></td>
       <td><code>org.jacoco.agent</code></td>
+      <td></td>
       <td>API to get a local copy of the agent</td>
     </tr>
     <tr>
       <td><code>org.jacoco</code></td>
+      <td><code>org.jacoco.agent</code></td>
+      <td><code>runtime</code></td>
+      <td>Agent</td>
+    </tr>
+    <tr>
+      <td><code>org.jacoco</code></td>
       <td><code>org.jacoco.ant</code></td>
+      <td></td>
       <td>Ant Tasks</td>
     </tr>
     <tr>
       <td><code>org.jacoco</code></td>
+      <td><code>org.jacoco.ant</code></td>
+      <td><code>nodeps</code></td>
+      <td>Ant Tasks <i>(all dependencies included)</i></td>
+    </tr>
+    <tr>
+      <td><code>org.jacoco</code></td>
+      <td><code>org.jacoco.cli</code></td>
+      <td></td>
+      <td>Command Line Interface</td>
+    </tr>
+    <tr>
+      <td><code>org.jacoco</code></td>
+      <td><code>org.jacoco.cli</code></td>
+      <td><code>nodeps</code></td>
```

Comment:
@Godin Some arguments to keep `nodeps` I can think of:

- This is consistent to the classifier for the Ant module. 
- Deploying shaded JARs to Maven is violation of best practice. So we have a explicit warning.
- If CLI would offer a API one day we can publish a un-shaded version without classifier.